Civi/ActionSchedule/RecipientBuilder.php

```diff
@@ -256,6 +258,7 @@ protected function buildAddlRepeatPass() {
         ->merge($this->prepareAddlFilter('c.id'), ['params'])
         ->where("c.is_deleted = 0 AND c.is_deceased = 0")
         ->groupBy("reminder.contact_id")
+        ->having("SUM(ISNULL(reminder.action_date_time)) = 0")
```

Comment:
For this one, the grouping here is only on contact id, so would this effectively require ALL the reminders for this contact to be sent already? E.g. if there was one for activities and one for events.